spec/factories/lists.rb

```diff
@@ -4,5 +4,11 @@
 
     title "A list"
     content "This is content"
+
+    trait :private do
+      private 'true'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.